infrastructure/main.tf

```diff
@@ -1,3 +1,11 @@
 provider "azurerm" {
   features {}
 }
+
+data "azurerm_resource_group" "example" {
+  name = "pip-ss-sbox-rg"
+}
+
+output "id" {
+  value = data.azurerm_resource_group.example.id
+} 
```

Comment:
what is all this for?

Reply:
not needed, was part of testing. will remove